test/e2e/scheduling/preemption.go

```diff
@@ -422,12 +422,27 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 		node, err = cs.CoreV1().Nodes().UpdateStatus(nodeCopy)
 		framework.ExpectNoError(err)
 
+		// print out existing priorities
```

Comment:
Could you please add a TODO to remove this after debugging is done?

Reply:
Absolutely. Done.

Reply:
Actually I would prefer not to merge temporary code like this. Instead, how about we wrap it in a block like
```
 if CurrentGinkgoTestDescription().Failed { }
```
and move it to a `JustAfterEach` thingy?

ref http://onsi.github.io/ginkgo/#separating-creation-and-configuration-justbeforeeach

Reply:
Emm... As this will be removed right after resolving the issue. Maybe not worth adding extra logic?

Reply:
I like Jonathan's idea. It is not really much additional logic. It is just a little bit of reorganization of the code. With that change, we could even leave the code and not remove it later.

Reply:
np, I will figure out. In that case, I don't need to comment it as "TODO", I suppose.